### DISCUSSION TITLE
[Feature] Allow modification of metadata from web UI

### DISCUSSION THREAD

**Discussion by grasponcrypto** [immich-app/immich#3991] [Feature Request]
### The feature

I have many pictures which are scanned copies of pictures pre-digital era.  These pictures get the metadata date from when they were scanned in, and I dont see an easy way to adjust the date or adding a location.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of  #1696
